File: JuanMorales_GenerarTiempos.py
import time
import numpy as np

def fibonacci(N):
	if N<=1:
		return N
	else:
		return fibonacci(N-1)+fibonacci(N-2)

def get_time(N):
	t0 = time.time()
	fibonacci(N)
	t1 = time.time()-t0
	return t1

lista=np.linspace(0,35,35)
for j in range (0,36):
	print j,",",get_time(j)

File: JuanMorales_GenerarTiempos.cpp
#include <iostream>
#include <typeinfo>
#include <math.h>
#include <cstdlib>
#include <ctime>
#include <iostream>
#include <fstream>

using namespace std;
#define n 35
float get_time(int N);
int fibonacci(int No);

int main(){
	for (int j=0;j<=n;j=j+1){
		cout <<j<<","<<get_time(j)<<"\n";
	}
	return 0;
}

int fibonacci(int N){
	if (N<=1){
		return N;
	}
	else{
		return fibonacci(N-1)+fibonacci(N-2);
	}
}

float get_time(int No){
	clock_t t;
	t = clock();
	fibonacci(No);
	t = clock() - t;
	float time = ((float)t)/CLOCKS_PER_SEC;
	return time;
}










